app/controllers/hosts_controller.rb

```diff
@@ -560,6 +560,7 @@ def process_hostgroup
               Host.new(params[:host])
             end
     @host.set_hostgroup_defaults
+    @host.set_compute_attributes
```

Comment:
No, If I change the compute_profile manually from UI, I still want to reset the attributes.